management-core/src/test/java/com/ns/task/services/ProductCoreServiceImpTest.java

```diff
@@ -78,6 +81,21 @@ public void insertProduct() {
         assertProduct(returned, productEntity);
     }
 
+    @Test
```

Comment:
Same issue that in `ProductRepositoryImplTest.java`

Reply:
Fixed